### PR TITLE
Add script to place call via Apple's Cellular Calls feature

### PR DESCRIPTION
**Pull request by alexjsteffen**
Add a basic script using the native Cellular Call Feature provided by Apple. There is no easy way to make a normal telephone call other than using the Contacts app or FaceTime (although the UX for this feature has gotten worse in Monterey. 

See: https://support.apple.com/en-ca/HT209456 

## Description

This script uses the "iPhone Cellular Call Feature" provided by apple to quickly make telephone calls without having to go via Contacts. Added one second of sleep to allow an individual to consider whether it was a mistake. It happens.

## Type of change

- [ ] New script command

## Screenshot

As simple as can be:

Demo

## Dependencies / Requirements

None. 

## Checklist

- [ ] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md) - Yes

### CODE REVIEW

File: commands/communication/call-with-iphone.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Call with iPhone
+# @raycast.mode compact
+
+# Optional parameters:
+# @raycast.icon 📱
+# @raycast.argument1 { "type": "text", "placeholder": "+1 514 555 1212" }
+# @raycast.packageName Call
```
Comment (unnamedd) [inline commands/communication/call-with-iphone.sh:11]:
```suggestion
# @raycast.packageName Communication
```

File: commands/communication/call-with-iphone.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+# Documentation:
+# @raycast.description Place a telephone call via your iPhone on wifi.
```
Comment (unnamedd) [inline commands/communication/call-with-iphone.sh:14]:
```suggestion
# @raycast.description Place a telephone call via your iPhone on Wi-Fi.
```

**Review by unnamedd — CHANGES_REQUESTED** [review]
Hey @alexjsteffen,

I made two really small suggestions in the code, nothing to worry about but it will make your Script Command more conformant to the others we have.

Thanks a lot for your first contribution on our community and welcome aboard!

File: commands/communication/call-with-iphone.sh (hunk 2 above)
Comment (alexjsteffen) [reply commands/communication/call-with-iphone.sh:14]: Good suggestions. Just merged your change. Cheers :)

Comment (unnamedd) [reply commands/communication/call-with-iphone.sh:14]: @alexjsteffen would apply this suggestion here as well? After it we will be good to go! ;)

**Comment by alexjsteffen** [comment]
Done. Have a great eve.

**Review by unnamedd — APPROVED** [review]
Now we are good to go! ✅ 
Thanks a lot for your contribution @alexjsteffen!

**Pull request merged**